CMSgov/design-system
# [WNMGDS-1645] component content migration pt 1

## Summary

* Migrating content for
  * Component overview
  * Badge
  * Autocomplete
  * Alert
  * Accordion
* Updated the rendering of headers to accommodate headings in `<code>` blocks

## How to test

`yarn build-storybook:gatsby` to build storybook once
`yarn build:gatsby && yarn serve:gatsby` to see doc site locally
